src/scores.ts

```diff
@@ -2,7 +2,7 @@ import { createHash } from 'crypto';
 import events from 'events';
 import snapshot from '@snapshot-labs/strategies';
 import { get, set } from './aws';
-import { paginateStrategies, sha256 } from './utils';
+import { paginateStrategies } from './utils';
```

Comment:
We could use sha256 here https://github.com/snapshot-labs/snapshot-score/pull/237/files#diff-5bcad33ffda7ca294c0f9f823153603ed41f2142a3290946e9595bcc85ae5649R71